README.md

```diff
@@ -338,6 +338,7 @@ With PyTorch/XLA we provide a set of performance profiling tooling and auto-metr
 * [Official tutorial](https://cloud.google.com/tpu/docs/pytorch-xla-performance-profiling-tpu-vm)
 * [Colab notebook](https://colab.research.google.com/github/pytorch/xla/blob/master/contrib/colab/pytorch-xla-profiling-colab.ipynb)
 * [Sample MNIST training script with profiling](https://github.com/pytorch/xla/blob/master/test/test_profile_mp_mnist.py)
+* [Utility script for capturing performance profiles](https://github.com/pytorch/xla/blob/master/scripts/capture_profile.py)
```

Comment:
Please add a reference on how to load the profiled filed on `tensorboard`.

Reply:
I suggest we port some of the comments in this [README](https://github.com/thisisalbertliang/training/pull/24/files) to PyTorch/XLA. I wonder if the `README` is the best place. Alternatively, we can consider another documentation guide on "Profiling" in PyTorch/XLA.

@JackCaoG wdyt?

Reply:
@miladm 

I am not sure where to add the tensorboard instructions. It seems like the [Official tutorial](https://cloud.google.com/tpu/docs/pytorch-xla-performance-profiling-tpu-vm) and [Colab notebook](https://colab.research.google.com/github/pytorch/xla/blob/master/contrib/colab/pytorch-xla-profiling-colab.ipynb) links in the README already teaches how to use tensorboard anyway?

For now, I have added instructions on how to run tensorboard in the module docstring for `capture_profile.py`. Let me know if you think there is a better location for these instructions.